tests/renderer_test.php

```diff
@@ -46,6 +51,7 @@ public function test_get_data() {
         $cmstype = new cms_types();
         $cmstype->set('name', 'somename');
         $cmstype->set('idnumber', 'test-somename');
+
         $cmstype->set('datasources', array_keys(dsbase::get_datasource_labels(false)));
```

Comment:
I think we should only care about built in datasources here which makes the renaming of `list -> userlist` all the more important since `dsbase::BUILTIN_DATASOURCES` uses `userlist`

Reply:
I have updated the statement to just include the builtins.